Sdk/Conversation/SwrveConversationContainerViewController.m

```diff
@@ -0,0 +1,31 @@
+#import "SwrveConversationContainerViewController.h"
+
+@interface SwrveConversationContainerViewController ()
+
+@property (nonatomic, retain) UIViewController* childController;
+@property (nonatomic) BOOL displayedChildrenViewController;
+
+@end
+
+@implementation SwrveConversationContainerViewController
+
+@synthesize childController;
+@synthesize displayedChildrenViewController;
+
+-(id) initWithChildViewController:(UIViewController*)child {
+    if (self = [super init]) {
+        self.childController = child;
+    }
+    return self;
+}
+
+- (void)viewDidAppear:(BOOL)animated
+{
+    [super viewDidAppear:animated];
+    if (!displayedChildrenViewController) {
+        displayedChildrenViewController = YES;
```

Comment:
viewDidAppear can be called multiple times. If for example a dialog appears or another view controller does, it will be called again later.